docs/organizing-and-exploring-results-in-the-ui/single-experiment-details.rst

```diff
@@ -0,0 +1,138 @@
+Exploring a single experiment
+=============================
+
+|Youtube Video|
+
+Click a line in the experiments table to see details of that experiment.
+
+
+   .. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/single_experiment.png
+      :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/single_experiment.png
+      :alt: Single experiment
+
+Inside the experiment, there are tabs in the left sidebar. Each displays specific content that is logged to Neptune for the specific experiment. Each tab has a unique URL.
+
+- **Charts**: All metrics (numeric type of data) are visualized as charts.
+- **Logs**: Logs are data that can be logged or tracked to the experiment. There are multiple types: Numeric, text, and images.
+- **Monitoring**: Displays information about utilization of the memory, CPU and GPU, as well as stdout and stderr.
+- **Artifacts**: Displays files uploaded to the experiment.
+- **Source code**: Displays sources uploaded to the experiment.
+- **Parameters**: Displays parameters uploaded to the experiment during experiment creation.
+- **Details**: Displays additional metadata information.
+
+For more information about the experiment data that is logged, see :ref:`Logging to Neptune <guides-logging-data-to-neptune>`.
+
+Organizing experiments charts
```

Comment:
[opinion] Maybe put video at the top. Right now, I read about "chart set", "Basic" atc, and I have no idea how it look in app.